bot.py

```diff
@@ -44,7 +47,7 @@ def move(white_team: teams.Team, bot_team: teams.Team, depth=2):
     if chess_utils.is_checkmated(bot_team, white_team) or chess_utils.is_tie(bot_team, white_team):
         return None
 
-    score, best_move = mini(white_team, bot_team, depth)
+    score, best_move = mini(white_team, bot_team, depth, -10000)
     print(best_move)
```

Comment:
Remove print if you don't think it's important. 
Also rethink all other prints in your code, you can leave them if you think they can be helpful or nice to have, maybe print turn number and score diff after every move.
But if there are prints used for debugging, they do shouldn't be in the final code.

Reply:
:+1: